modules/library/referencing/src/main/java/org/geotools/referencing/factory/OrderedAxisAuthorityFactory.java

```diff
@@ -224,8 +224,8 @@ private static boolean booleanValue(final Hints userHints, final Hints.Key key)
      * invoked by constructors only.
      */
     private void completeHints() {
-        hints.put(Hints.FORCE_STANDARD_AXIS_UNITS, Boolean.valueOf(forceStandardUnits));
-        hints.put(Hints.FORCE_STANDARD_AXIS_DIRECTIONS, Boolean.valueOf(forceStandardDirections));
+        hints.put(Hints.FORCE_STANDARD_AXIS_UNITS, forceStandardUnits);
+        hints.put(Hints.FORCE_STANDARD_AXIS_DIRECTIONS, forceStandardDirections);
```

Comment:
Can I confirm you are building in Java 8?

We have had on occasion added code like the above to get around differences between java 8 and java 11 compiler. Differences in the byte code produced has caused us to add extra casts into our code on occasion.

Reply:
Java 8 is my development environment for work.  And all my testing of the changes where done in Java 8.

Reply:
I can't resist following IntelliJ inspection recommendations.  It said this was not required.

Reply:
In this case it probably is not required, I just wanted to double check :)